fontbe/src/orchestration.rs

```diff
@@ -16,7 +19,7 @@ use crate::{error::Error, paths::Paths};
 #[derive(Clone, Debug, PartialEq, Eq, Hash, PartialOrd, Ord)]
 pub enum WorkIdentifier {
```

Comment:
Just below, we have a type called `AnyWorkId`. Preferring shorter names, I'd recommend renaming `WorkIdentifier` and `FeWorkIdentifier` to `WorkId` and `FeWorkId` respectively, but in any case, it would be nice if they were all consistent.

Reply:
Oops, will fix.